.githooks/post-merge

```diff
@@ -0,0 +1,7 @@
+function changed {
+  git diff --name-only HEAD@{1} HEAD | grep "^$1" > /dev/null 2>&1
+}
+
+if changed 'package-lock.json'; then
+  echo "📦 package-lock.json changed. Run npm install to bring your dependencies up to date."
```

Comment:
Why not check for yarn.lock and run yarn to update deps?

Reply:
Yeah, actually have that in a need to update version: https://github.com/patronage/spiegelworld/blob/main/website/.githooks/post-merge.sh

But it still wasn't working, flagged for a future issue #53